### ISSUE TITLE
pub.dev/api/documentation/fo_components broken

### ISSUE THREAD

**Issue by jonasfj** [dart-lang/pub-dev#4810]
```
2021-05-18 22:26:07.914 UTC+2
pub: Handling request: https://pub.dev/api/documentation/fo_components
Error
2021-05-18 22:26:44.410 UTC+2
api_builder: Unhandled error in API handler (incidentId: b7d7f0d7-44b9-4266-a6ca-0944e5fae071)

Error:
    Could not successfully look up all keys due to resource constraints.

Stack:
    #0      GrpcDatastoreImpl.lookup (package:appengine/src/grpc_api_impl/datastore_impl.dart:151:9)
    <asynchronous suspension>
    #1      StackZoneSpecification._registerUnaryCallback.<anonymous closure> (package:stack_trace/src/stack_zone_specification.dart)
    <asynchronous suspension>
     
Error
2021-05-18 22:26:44.411 UTC+2
pub: Request handler done. 
```

**Comment by jonasfj** [comment]
doesn't seem to be a new issue, and might have to do with the shear number of versions this package has.

**Comment by isoos** [comment]
This seems to happened after #4783 got merged and deployed. We should probably size-limit the content of the scorecards.

**Comment by isoos** [comment]
This has last happened on May 20. That release contained a new `dartdoc` version. Earler dartdoc runs for `package:fhir` may have contained a long `DartdocReport` and with the batched reads, it was out of the resource limits. Since then, we have introduced a hard limit of the generated reports (32 KiB), and with the current batch limit, the combined size of the `ScoreCard` entities returned in a single lookup should not be much larger than 640 KiB, certainly under 1MiB.
